vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -91,9 +96,25 @@ func main() {
 	}
 
 	healthCheck := metrics.NewHealthCheck(time.Minute)
-	metrics.Initialize(*address, healthCheck)
 	metrics_admission.Register()
 
+	go func() {
```

Comment:
Would it make sense to put this block into some shared util package?

Reply:
Done in 7ec04b330830836ed483572edc9c1dad3c087f38